src/dataMap/metaManager/metaSchema.js

```diff
@@ -1019,6 +1019,8 @@ export default () => {
 
     /**
      * Class name for the Handsontable container element.
+     * It can also be used to assign class names for table cells,
+     * for example, if it is provided in `cells` or `columns` options.
      *
      * @memberof Options#
```

Comment:
In this snippet, the first comment should be removed, because it is only half-true:

<img width="736" alt="Screenshot 2020-12-09 at 12 56 35" src="https://user-images.githubusercontent.com/566463/101626987-fabbb400-3a1d-11eb-96c2-7703260ade6f.png">

Reply:
Fixed in 84ddb44e0f8098a797640cc90262b961a4ca782b